src/qt/guiutil.cpp

```diff
@@ -768,23 +768,66 @@ QString formatDurationStr(int secs)
     return strList.join(" ");
 }
 
+void sortLocaleAware(QStringList &sList)
+{
+    std::sort(sList.begin(), sList.end(), [](const QString &s1, const QString &s2){
+        return s1.localeAwareCompare(s2) < 0;
+    });
+}
+
 QString formatServicesStr(quint64 mask)
 {
     QStringList strList;
 
     for (const auto& flag : serviceFlagsToStr(mask)) {
+        if (flag == "NETWORK_LIMITED"){
+        strList.append(QString::fromStdString("LIMITED"));
```

Comment:
If we're going to start renaming these, "recent-blocks" makes more sense than "LIMITED"...